crypto/jwt.go

```diff
@@ -0,0 +1,14 @@
+package crypto
+
+import jwt "github.com/dgrijalva/jwt-go"
```

Comment:
maybe use `gopkg.in/dgrijalva/jwt-go.v2` instead to not rely on a github repo here